common/config/rush/common-versions.json

```diff
@@ -42,7 +42,5 @@
      * For example, allow some projects to use an older TypeScript compiler
      * (in addition to whatever "usual" version is being used by other projects in the repo):
```

Comment:
You should make sure the React versions earlier in this file are up to date

Reply:
Thanks! I think I will change all of the `@types` to `>=` and also pin them in `common-versions.json` instead of pinning every `package.json`.